src/libs/tmdk/src/commands/deploy.ts

```diff
@@ -57,33 +58,31 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
   });
 
 function syncReadFile(path: string) {
-  const result = fs.readFileSync(path, "utf-8");
-  return `${result}`; // TODO more idiomatic
+  return fs.readFileSync(path, "utf-8");
 }
 
 export const handler = async (argv: Arguments<Options>) => {
-  const workspaceId = argv["workspace-id"] as string; // TODO allow it to be optional
-  const region = argv.region;
-  const dir = argv.dir;
-  const workspaceIdStr = `${workspaceId}`; // TODO idiomatic
+  const workspaceIdStr: string = argv["workspace-id"]; // TODO allow it to be optional (i.e. option to autogenerate workspace for them)
+  const region: string = argv.region;
+  const dir: string = argv.dir;
   console.log(
     `Deploying project from directory ${dir} into workspace ${workspaceIdStr} in ${region}`
   );
 
-  initDefaultAwsClients({ region: `${region}` }); // TODO idiomatic
+  initDefaultAwsClients({ region: region });
   // read tmdk json file
   const tmdk_config_buffer = fs.readFileSync(`${dir}/tmdk.json`, "utf-8"); // TODO encodings
```

Comment:
Not necessary for this pr but can we add a todo to error check the tmdk definition exists? Tmdk should not just throw readFileSync error if the user does not have tmdk definition

Reply:
Good catch, should be a small enough change I could add here